src/common/users.ts

```diff
@@ -118,6 +118,7 @@ export interface StreakRecoverQueryResult {
   canRecover: boolean;
   cost: number;
   oldStreakLength: number;
+  regularCost: number;
```

Comment:
Added this so we can control regular price from API side as well when free streak restore is shown